src/fs.rs

```diff
@@ -406,7 +407,16 @@ impl Fs {
 
         let bundle_guest_path = home_directory.join(&bundle_dir_name);
 
-        let documents_host_path = Path::new("touchHLE_sandbox")
+        // TODO: instead of path hardcoding, we should use https://wiki.libsdl.org/SDL2/SDL_AndroidGetInternalStoragePath
+        // but... it's not exposed via rust-sdl2 :(
```

Comment:
Can we have a single function somewhere for getting the path? It can be shared between this file and the file that loads fonts. I guess it might also be used for loading the default settings at some point.

If we want to use `SDL_AndroidGetInternalStoragePath` in that function, we can probably access it via the rust sdl2-sys crate.

Reply:
Done for a common function, i placed it as pub in the fs module

Reply:
Thanks, makes sense to me :)